### DISCUSSION TITLE
Odd toxin descriptions

### DISCUSSION THREAD

**Discussion by monomethylhydrazine** [FakeFishGames/Barotrauma#9498] [Ideas and suggestions]
Morbusine, cyanide, radiotoxin, and sufforin are all said to be neurotoxins from their ingame description even though (at least cyanide and radiotoxin) that's just wrong.
Cyanide (in real life) affects ribosomes (components affecting cellular protein production), radiotoxin is likely just highly absorbable radioactive material, and (judging by its affliction), morbusine likely affects the muscles of the cardiovascular system and pulmonary system rather than the brain.
Sufforin, given its infliction of delusions, may be a neurotoxin but that's up to developer discretion.
My suggestion is to give more accurate descriptions to the rest of the toxins, such as by stating that cyanide "shuts down cellular processes" or morbusine "affects the lungs and heart" to give better clues as to the specific affects of the toxin.

**Comment by Thirty-Percent** [comment]
Having credible / scientifically logical (if not realistic, of course) descriptions would be a big plus I think.

**Comment by Ek-Videogames** [comment]
After the switch from 'Legacy' to the new medical system, the 3 poisons _(other than radiotoxin)_ became pretty much identical reskins of eachother. Adding mechanical differences between them again is something that also would need to be addressed at some point when it comes to defining their differences.
